### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
created the home page for e-commerce 
Please review and give your comments

You can have a preview here: https://sona-fancy-homepage.netlify.app/#

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
Looks good

File: css/button.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.button-main{
+    padding: 5px 10px 5px 10px;
+    border-radius: 10px;    
+    margin-left: 1rem;
+    margin-right: 1rem;
```
Comment (Naveenchandar) [inline css/button.css]: Please use shorthand property for margin.

File: css/master.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  }
+  
+@media screen and (max-width: 850px) {
+    div.card_resize{
+        width:150px;
+        height:250px;        
+    }
+    div.home_main_content {
+        font-size: 20px;
+    }
```
Comment (Naveenchandar) [inline css/master.css:39]: any reason for using px instead of relative units ?

File: home.html [hunk 3]
```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <script src="https://kit.fontawesome.com/c2fdf537fc.js"
```
Comment (Naveenchandar) [inline home.html:8]: No need of using script tag for font awesome. It can be done using link tag.

File: variable.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+:root{
+    --grey-color: #9ca3af;
+    --black-color: #000000;
+    --white-color: #ffffff;
+    --red-color: red;
```
Comment (Naveenchandar) [inline variable.css]: Please use hex codes for consistency

File: css/button.css (hunk 1 above)
Comment (NandhiniKrishnamoorthy) [reply css/button.css]: Updated the values

File: variable.css (hunk 4 above)
Comment (NandhiniKrishnamoorthy) [reply variable.css]: Updated to hex codes

File: home.html [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+                </div>
+            </div>
+            
+            <div class="text-center">
+                <ul class=" list_style py-1">
+                   <a><li class="list_inline px-1">Home</li></a>
+                   <a><li class="list_inline px-1">Earrings</li></a>
+                    <a><li class="list_inline px-1">Neck Wear</li></a>
+                    <a><li class="list_inline px-1">Bangles</li></a>
+                    <a><li class="list_inline px-1">Others</li></a>
```
Comment (surbhi84) [inline home.html:66]: put anchor inside the li tag, li should be the direct child of list.

File: homepage/style/badge.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    border-radius: 50%;
+    border: var(--white-color) 1px solid; 
+    font-size: 1.25rem;
+    font-weight: 600;
+    padding: 1px 5px;
+    text-align: center;
+    line-height: 1;
+    position: absolute;
+    left: 100%; 
+    top: -0.1em;
```
Comment (surbhi84) [inline homepage/style/badge.css:22]: here top:0 ; would do the same thing, also try to stick to one either rem or em.

**Review by surbhi84 — APPROVED** [review]
added comments make changes accordingly, rest looks good to me.

**Pull request merged**